src/components/report/CharGrid.js

```diff
@@ -0,0 +1,34 @@
+import VariableControls from "./VariableControls.vue";
+
+export default {
+	props: ["font"],
+	components: {
+		VariableControls
+	},
+	data() {
+		return {
+			variableStyles: "",
+			chars: this.font.supportedCharacters,
+			charCount: this.font.supportedCharacters.length,
+			hasAxes: this.font.variable && this.font.variable.axes.length,
+			showAxes: this.font.isVariable,
+			hasFeatures: false, // TODO: Should be determined by the font
+			showFeatures: false // TODO: Should be determined by the font
+		};
+	},
+	methods: {
+		entitify: function(code) {
+			return `&#x${code};`;
+		},
+		toggleAxes: function() {
+			this.variableStyles = "";
+			this.showAxes = !this.showAxes;
+		},
+		// toggleFeatures: function() {
```

Comment:
This is used in the template but commented out?

Reply:
It's commented out there as well. I'd like to keep it around as I'll be wiring it up later today/tomorrow! :-)